### PR TITLE
Initial Commit - Homework 1

### PR DESCRIPTION
**Pull request by moorphy**
# Week 1 HW Submission

Please fill out the information below in order to complete your assignment. Feel free to update this comment later if necessary.

* Comfort rating on this assignment (1-5):
2.5. I am picking it up, but still need more reps.
* Completion rating on this assignment (complete/incomplete):
complete. I think. ;)
* Github handles of code I've reviewed:
none.

### CODE REVIEW

**Comment by moorphy** [comment]
The last commit passed the check.

File: package.json [hunk 1]
```diff
@@ -6,11 +6,15 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moorphy/week1"
+  },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "eslint": "^4.18.0",
+    "eslint": "^4.19.1",
```
Comment (bhague1281) [inline package.json:17]: 👍 😄

File: src/components/DivElement.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
-
-// Export class here
-module.exports = {};
+module.exports = class DivElement extends HTMLElement {
```
Comment (bhague1281) [inline src/components/DivElement.js:4]: Nice job exporting + declaring the class in one line.

**Review by bhague1281 — APPROVED** [review]
Great job!

**Pull request closed without merging**